### PR TITLE
DOCS-7787: Document isMaster.lastWrite

### PR DESCRIPTION
**Pull request by i80and**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/2797)

### CODE REVIEW

**Comment by kay-kim** [comment]
@i80and  -- Is there somewhere that this is staged?  I just wanted to see how the include file reads in its context.

**Comment by i80and** [comment]
https://docs-mongodbcom-staging.corp.mongodb.com/andrew/DOCS-7787/reference/command/isMaster.html#dbcmd.isMaster

**Comment by kay-kim** [comment]
Not sure if this flows quite right:
https://docs-mongodbcom-staging.corp.mongodb.com/andrew/DOCS-7787/reference/command/replSetGetStatus.html#replSetGetStatus.optimes

**Comment by i80and** [comment]
Eesh, you're right; that file's change was an after-thought, and it shows.

Fixed, but let me know if you want me to do another round of review.

File: source/reference/glossary.txt [hunk 1]
```diff
@@ -585,6 +585,18 @@ Glossary
       basic mechanism enabling :term:`replication` in MongoDB.
       See :doc:`/core/replica-set-oplog`.
 
+   optime
+      .. versionchanged:: 3.2
+
+         MongoDB 3.2 introduced replication
+         :rsconf:`protocolVersion: 1 <protocolVersion>` with a new optime
+         format.
```
Comment (i80and) [inline source/reference/glossary.txt]: I see. How about

    The following describes the optime format used by `protocolVersion: 1`, introduced by MongoDB 3.2.

Comment (i80and) [inline source/reference/glossary.txt]: We use "introduced in" in our existing corpus, and is generally a more common form I feel. Updated.

File: source/reference/command/replSetGetStatus.txt [hunk 2]
```diff
@@ -195,23 +195,13 @@ following fields:
...
-   progress for various operations.
-   
-   If using :rsconf:`protocolVersion: 1 <protocolVersion>`, the optime
-   value is a document that contains:
-
-   - ``ts``, the :ref:`Timestamp <document-bson-type-timestamp>` of
-     the operation.
+   The ``optimes`` field holds a document that contains
+   :term:`optimes <optime>` showing replication progress for various
+   types of operations.
```
Comment (i80and) [inline source/reference/command/replSetGetStatus.txt]: As far as I can see, there are three main differences:

- s/information on various optimes/optimes
- s/reported here to show/showing
- No longer notes different behavior with protocol version 0

The first is only incorrect if the following documentation is also wrong: the documented document fields are just optimes.

Comment (kay-kim) [inline source/reference/command/replSetGetStatus.txt]: So, it's the second point:
optimes showing replication progress ...  makes it sound like there are optimes that show one thing and other optimes showing some other optime.

Also, separately, both the original wording and new version for "various operations" and "various types of operations" -- I wonder if we should just get rid of it as  it sounds like there are different optimes for updates or something.

**Comment by kay-kim** [comment]
merged!

**Pull request closed without merging**